src/pages/home/components/About.js

```diff
@@ -1,28 +1,37 @@
 import React from "react";
 import { Box } from "@mui/material";
-import about_image from "assets/about_2.png";
+import about_image from "assets/about/about.png";
+import about_mobile from "assets/about/about_mobile.svg";
 import sunshine_image from "assets/sunshine.svg";
 import { useNavigate } from "react-router-dom";
 import HomeButton from "shared/components/HomeButton";
+import { useMediaQuery } from "react-responsive";
 
-const typographyStyle = "text-[18px] leading-tight font-proxima-nova my-5 w-full pr-12";
+const typographyStyle = "text-[12px] sm:text-[15px] xl:text-[20px] leading-tight font-proxima-nova my-5 w-full sm:pr-6";
 
 const About = () => {
   const navigate = useNavigate();
-
+  const isMobile = useMediaQuery({ query: `(max-width: 639px) ` });
   const handleClickButton = () => {
     navigate("about");
   };
 
   return (
-    <Box className={`flex justify-between items-center py-14`}>
-      <Box className="w-1/2 h-full">
-        <Box className={`w-full flex pb-1.5`}>
-          <span className="text-3xl font-bold font-oswald text-primary pt-8">
-            WHO WE ARE
-          </span>
-          <img src={sunshine_image} alt="Sunshine" className="relative -z-10 right-4"></img>
-        </Box>
+    <Box className="flex flex-col w-full text-center sm:text-left justify-center items-center sm:items-start pt-7 sm:pt-20">
+      <Box className="w-full sm:w-1/2 flex justify-center sm:justify-start pb-4 sm:pb-1.5 pl-8 sm:pl-0">
+        <span className="text-base sm:text-3xl xl:text-4xl font-bold font-oswald text-primary pt-4 sm:pt-6">
+          WHO WE ARE
+        </span>
+        <img src={sunshine_image} alt="Sunshine" className="h-6 sm:h-10 xl:h-full relative right-2 top-1 sm:top-2 xl:top-0 "></img>
+      </Box>
+      <a href="/about" className="sm:w-1/3 2xl:w-fit sm:absolute sm:right-1/10">
+        <img
+          src={isMobile ? about_mobile : about_image}
+          alt="GISAU Executives"
+          className="w-full"
+        />
+      </a>
+      <Box className="w-[60%] sm:w-1/2 h-full">
         <p className={typographyStyle}>
           Hello! We are a non-profit cultural organization centered in promoting
```

Comment:
```suggestion
          Halo! We are a non-profit cultural organization centered in promoting
```

Reply:
Are we sure we want to use Halo? 
<img width="679" alt="image" src="https://user-images.githubusercontent.com/65900990/218374434-23e6abb3-d079-4344-b9b1-62f24c7bccdd.png">

Reply:
@jenaarianto any thoughts?? I feel like we need to include an introduction in bahasa indo so Halo should b fine.

Reply:
Changed to Halo after a thorough discussion